tests/test_views.py

```diff
@@ -0,0 +1,27 @@
+from django.urls import reverse
+from django.test import modify_settings, TestCase
+
+from .factories import ArticleFactory
+
+
+class TestView(TestCase):
+    slug_name = 'ellobiidae'
+
+    @modify_settings(INSTALLED_APPS={'append': ['tests']})
+    def test_index_response_ok(self):
+        response = self.client.get(reverse('article_index'))
+        self.assertEqual(response.status_code, 200)
+
+    @modify_settings(INSTALLED_APPS={'append': ['tests']})
```

Comment:
Do you not want `tests` in INSTALLED_APPS all the time?  Might as well just add it to `run.py`?